app/apps/client/admin/apps.html

```diff
@@ -75,6 +75,7 @@
 													{{app.description}}
 												{{/if}}
 											</span>
+											{{!-- There is no current way to get the app categories from the API --}}
```

Comment:
https://marketplace.rocket.chat/v1/categories :)

Reply:
@tassoevan can you explain what you mean here? We have an API for this and each App Information has an array of categories it belongs to.